src/detect-filesha1.c

```diff
@@ -103,55 +100,35 @@ static int SHA1MatchLookupString(ROHashTable *hash, const char *string)
     uint8_t sha1[20];
     if (ReadHashString(sha1, string, "file", 88, 40) == 1) {
         void *ptr = ROHashLookup(hash, &sha1, (uint16_t)sizeof(sha1));
-        if (ptr == NULL)
-            return 0;
-        else
-            return 1;
+        PASS_IF(ptr != NULL);
     }
-    return 0;
+    FAIL;
 }
```

Comment:
Hi,
I was unsure if I should make a change here as this seems to be more of
a helper test function rather than a unit test like `SHA1MatchTest01()`. However, I
made the change as it was included under `ifdef UNITTESTS`. Let me know if it is
wrong.